project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
Fixing it in Jetty sounds like it's probably a good thing to do.  How would you propose doing that - by adding it to the top-level `:dependencies` in the tk-jetty9 "project.clj"?